src/riak_kv_delete.erl

```diff
@@ -101,6 +100,19 @@ delete(ReqId,Bucket,Key,Options,Timeout,Client,ClientId,VClock) ->
             end
     end.
 
+new_maybe_ts_object(Bucket, {PK, LK}) ->
+    %% TS objects have this peculiar way to use the two keys: time
+    %% quanta-based partition key, for calculating coverage; and local
+    %% key for actual lookups:
+    RO_ = riak_object:new({Bucket, Bucket}, PK, <<>>),
```

Comment:
I got confused reading this with `RO_` as it is and `_RO` as I mistakenly read it meaning it is going to be an unused binding.  Should be `RO` and `MD1`, `MD2`.